ui/components/app/nft-details/__snapshots__/nft-details.test.js.snap

```diff
@@ -56,12 +56,14 @@ exports[`NFT Details should match minimal props and state snapshot 1`] = `
       >
         <div>
           <h4
-            class="box box--margin-top-1 box--margin-bottom-2 box--flex-direction-row typography typography--h4 typography--weight-bold typography--style-normal typography--color-text-default"
+            boxprops="[object Object]"
+            class="box mm-text mm-text--heading-sm mm-text--font-weight-bold box--flex-direction-row box--color-text-default"
           >
             MUNK #1
           </h4>
           <h5
-            class="box box--margin-top-1 box--margin-bottom-4 box--flex-direction-row typography typography--h5 typography--weight-normal typography--style-normal typography--color-text-muted typography--overflowwrap-break-word"
+            boxprops="[object Object]"
```

Comment:
Looking really good. But you need to update the snapshot here since it should not have `boxprops` for the `Text `component

Reply:
@NidhiKJha ,
Kindly check.